entry_tasks/89_sequence_type_check/main.py

```diff
@@ -0,0 +1,49 @@
+def parse_value(string_value):
+    if string_value.isdigit():
+        return int(string_value)
+    elif string_value.strip().replace('.', '').isdigit():
+        return float(string_value)
+    elif string_value.strip().lower() == "false":
+        return False
+    elif string_value.strip().upper() == "False":
+        return False
+    elif string_value.strip().lower() == "true":
+        return True
+    elif string_value.strip().upper() == "True":
+        return True
+    elif string_value.strip().lower() == "none":
+        return None
+    elif string_value.strip().upper() == "None":
+        return None
+    elif string_value.__str__():
```

Comment:
not following what you are actually want to check here?

if string_value is not empty - you can just check:

```py
elif string_value:
    ....
```